src/Header/index.tsx

```diff
@@ -0,0 +1,45 @@
+import AppBar from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+import SearchIcon from "@mui/icons-material/Search";
+import "./style.scss";
+import logo from "./logo.svg";
+
+export default function Header() {
+  const displayDesktop = () => {
+    return (
+      <Toolbar className="header">
+        <div className="menu-container">
+          <a href="/">
+            <img className="logo" src={logo} alt="" />
+          </a>
+          <ul className="menu">
+            <li className="menu-item">
+              <a href="/">Movies</a>
+            </li>
+            <li className="menu-item">
+              <a href="/">TV Shows</a>
+            </li>
+            <li className="menu-item">
+              <a href="/">People</a>
+            </li>
+            <li className="menu-item">
+              <a href="/">More</a>
+            </li>
```

Comment:
please move to separete component and use iterator for it

Reply:
Separate component done, but failed to do the iterator